src/js/clinvar/eva-clinical-widget-panel.js

```diff
@@ -540,6 +540,9 @@ EvaClinicalWidgetPanel.prototype = {
 
         var newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + '?' + 'Clinical Browser&' + $.param(values);
         history.pushState ('forward', '', newurl);
+        //sending tracking data to Google Analytics
+        ga('send', 'event', { eventCategory: 'Search', eventAction: 'Clinical Browser', eventLabel: $.param(values)});
```

Comment:
Is there any criteria (described in a tutorial or something) to assign 'eventCategory' and 'eventAction'? To me, it looks like these 2 should be swapped: an action "search" is performed in a category "Clinical Browser", not the opposite.